aspnetcore/signalr/configuration.md

```diff
@@ -162,6 +162,31 @@ let connection = new signalR.HubConnectionBuilder()
     .build();
 ```
 
+::: moniker range=">= aspnetcore-3.0"
+
+Instead of a `LogLevel` value, you can also provide a `string` value representing a log level name. This is useful when configuring SignalR logging in environments where you don't have access to the `LogLevel` constants.
+
+```javascript
+let connection = new signalR.HubConnectionBuilder()
+    .withUrl("/myhub")
+    .configureLogging("warn")
+    .build();
+```
+
+The below table lists the available log levels. The value you provide to `configureLogging` sets the **minimum** log level that will be logged. Messages logged at this level, **or the levels listed below it on the table**, will be logged.
+
+| `string` Name | `LogLevel` value |
```

Comment:
Suggest removing code formatting from headings; with it they don't stand out as headings very well.  The headings could be just string and LogLevel.
```suggestion
| string | LogLevel |
```